keyboards/keebfront/vanguard65/info.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
Please run `qmk format-json` on this file.

Reply:
@fauxpark Both changes handled. Thanks!